HHS/Head-Start-TTADP
# [TTAHUB-3478] Goal nudge version 2

## Description of change
This PR updates the "goal nudge" flows to not spawn modals or change context on the user, instead moving through steps based on the goal's status and state

## How to test
Go to the RTR and create a new goal
Confirm that each flow matches the mockup flow (linked in the Jira)
- In Progress or Not Started goal suggested by the nudge
- Suspended goal, suggested by nudge
- Closed goal, suggested by nudge
- OHS standard goal
- Ignore all suggestions and create a new goal

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3478


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
